atlp-rwanda/Team-furebo-E-commerce-bn
# #184651343 implement functionality for viewing specific items as a buyer or seller

### **What does this PR do?**
A User should be able to view a specific item
##### **Description of Task to be completed?**
As a User
I should be able to retrieve a given item.
  **As a Seller** I should be able to see an item if it is from my 
   collection

  **As a Buyer**, I should be able to see any item from the available items.

#### How should this be manually tested?
- First Clone the repository  
- Navigate to the branch project 
- use **`npm install`** to install the required dependencies.
- to test use **`npm run test`** and see the coverage.
- use **`npm start`** to start the server
- You can log in as a seller to list all items in your collection
- Or you can log in as a buyer to see all available products

#### What are the relevant pivotal tracker/Trello stories?
[#184651324](https://www.pivotaltracker.com/n/projects/2633635)
